app/models/concerns/products/scheduling_support.rb

```diff
@@ -41,6 +41,33 @@ def next_available_reservation(after: Time.zone.now, duration: 1.minute, options
     reservation_in_week(after, duration, rules, options)
   end
 
+  def quick_reservation_data(after: 1.minute.from_now)
```

Comment:
`1.minute.from_now` is used here only because specs are frozen in time, and is not otherwise useful.

Reply:
how does the time freeze cause issues?

Reply:
I want to make sure the spec failure isn't telling us something important

Reply:
@giladshanan the issue was with `Reservations::Validations#in_the_future?`

`ReservationCreator#save` runs `ReservationCreator#save_reservation_and_order_detail`, which produces the error "Validation failed: Reserve Start must be in the future"